test/liquid_voting_web/absinthe/mutations/create_delegation/existing_delegations_test.exs

```diff
@@ -22,48 +22,59 @@ defmodule LiquidVotingWeb.Absinthe.Mutations.CreateDelegation.ExistingDelegation
             email
             name
           }
+          id
         }
       }
       """
 
-      {:ok, %{data: %{"createDelegation" => original_delegation}}} =
-        Absinthe.run(query, Schema, context: %{organization_id: @organization_id})
+      {:ok, %{data: %{"createDelegation" => updated_delegation}}} =
+        Absinthe.run(query, Schema, context: %{organization_id: global_delegation.organization_id})
 
-      assert original_delegation["delegator"]["email"] == @delegator_email
-      assert original_delegation["delegate"]["email"] == @delegate_email
+      assert updated_delegation["delegator"]["email"] == global_delegation.delegator.email
+      assert updated_delegation["delegate"]["email"] == another_delegate.email
+      assert updated_delegation["id"] == global_delegation.id
+    end
+  end
+
+  describe "create global delegation when proposal delegation to same delegate already exists" do
+    test "returns the new global delegation" do
+      proposal_delegation_1 = insert(:delegation_for_proposal)
```

Comment:
`proposal_delegation`, no need for the `_1`

Reply:
done